lxc/storage.go

```diff
@@ -88,6 +88,10 @@ lxc storage delete [<remote>:]<pool>
 lxc storage edit [<remote>:]<pool>
     Edit storage pool, either by launching external editor or reading STDIN.
 
+lxc storage <pool-name> btrfs.mount_options "<mount option>,<mount option>,<mount option>"
+    add btrfs mount options to btrfs storage pool
+    EXAMPLE: lxc storage <pool-name> btrfs.mount_options "rw,nospace_cache,lazytime,user_subvol_rm_allowed,autodefrag"
+
```

Comment:
running the `make i18n` command (from within the cloned lxd folder' I get this

`go get -v -x github.com/snapcore/snapd/i18n/xgettext-go/
make: go: Command not found
Makefile:98: recipe for target 'update-pot' failed
make: *** [update-pot] Error 127`

I know this is not a `make` workshop here. just wanted to notify you. 
Trying to solve this matter with god and googles help now and then proceed with the matter